web-ng/src/app/components/inline-editor/inline-editor.component.scss

```diff
@@ -14,12 +14,33 @@
  * limitations under the License.
  */
 
+@import 'variables';
+
 :host {
-  display: block;
-  font-size: 1.5em;
-  margin-top: 0.83em;
-  margin-bottom: 0.83em;
-  margin-left: -8px;
-  margin-right: 0;
   font-weight: bold;
+  font-size: 14pt;
+}
+
+// Share styling of inline edit with inline-edit-title and inline-editor component.
+.inline-editor-input {
+  font-family: inherit;
+  font-size: inherit;
+  border: 2px solid transparent;
+  outline: none;
+  border-radius: 3px;
+  padding: 2px 1px 2px 4px;
+}
+
+// !important used to override hover border color.
+.inline-editor-input:focus {
```

Comment:
Does `:host:focus` and `:host:hover` work? If not please ignore previous comment.